dts/bindings/i2c/ti,tca9546a.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2020, Innoseis BV
+# SPDX-License-Identifier: Apache-2.0
+
+description: TCA9546a I2C Mux
```

Comment:
I think it would be nice to give an example for how to put a sensor device underneath this thing here using a multiline comment in the description.

Reply:
What do you mean by a multiline comment ?

Reply:
LIke here https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/i2c/nordic%2Cnrf-twis.yaml#L4 ?